app/src/main/java/ayds/spotisong/aurora/otherdetails/model/OtherInfoModelImp.kt

```diff
@@ -0,0 +1,25 @@
+package ayds.spotisong.aurora.otherdetails.model
+
+import android.content.Context
+import ayds.spotisong.aurora.otherdetails.model.repository.OtherInfoRepository
+import ayds.spotisong.aurora.otherdetails.model.repository.local.DataBase
+import ayds.spotisong.aurora.otherdetails.presenter.OtherInfoPresenter
+import ayds.spotisong.aurora.song.model.Song
+
+class OtherInfoModelImp(repository: OtherInfoRepository, presenter: OtherInfoPresenter) : OtherInfoModel{
```

Comment:
no hace falta definir estas propiedades, se pueden definir directamente en el constructor
```suggestion
class OtherInfoModelImp(private val repository: OtherInfoRepository, private val presenter: OtherInfoPresenter) : OtherInfoModel{
```